libc-test/build.rs

```diff
@@ -141,6 +141,7 @@ fn main() {
     }
 
     if linux || android {
+        cfg.header("linux/perf_event.h");
```

Comment:
In the past we've been burned by including `linux` headers (mismatches with userspace or otherwise using unstable bits), is this appropriate to define?